### PR TITLE
fixes a makefile target to run an integration test for checking encryption

### PR DESCRIPTION
**Pull request by p0lyn0mial**
it looks like the test was never exercised in CI

### CODE REVIEW

**Review by ravisantoshgudimetla — COMMENTED** [review]
I noticed this error when I loaded the project in GoLand too and was about to provide a fix. Thank you for fixing it @p0lyn0mial 

/lgtm

/hold 

for auth folks to give final approval

File: test/e2e-encryption/encryption_test.go [hunk 1]
```diff
@@ -360,7 +374,7 @@ func TestEncryptionIntegration(tt *testing.T) {
 	deployer.DeleteOperandConfig()
 	waitForConfigs(
 		// 7 is migrated and hence only one needed, but we rotate through identity
-		"kubeapiservers.operator.openshift.io=identity,aescbc:7;kubeschedulers.operator.openshift.io=identity,aescbc:7",
+		"kubeapiservers.operator.openshift.io=identity,aescbc:7,aescbc:5;kubeschedulers.operator.openshift.io=identity,aescbc:7,aescbc:5",
```
Comment (p0lyn0mial) [inline test/e2e-encryption/encryption_test.go:377]: since `aescbc:5` wasn't deleted it should be on the list, right?

**Pull request closed without merging**